--- camwire2-1.9.0-Source/camwire/camwire.c.orig	2011-02-17 03:31:12.000000000 +0100
+++ camwire2-1.9.0-Source/camwire/camwire.c	2011-08-06 18:12:59.946406902 +0200
@@ -3633,7 +3633,7 @@
 			   &capability));
     if (feature_is_usable(&capability))
     {
-	if (capability.trigger_polarity == DC1394_TRUE)
+	if (capability.trigger_polarity == DC1394_TRIGGER_ACTIVE_HIGH)
 	    set->trigger_polarity = 1;
 	else
 	    set->trigger_polarity = 0;
@@ -3980,7 +3980,6 @@
     dc1394video_mode_t video_mode;
     dc1394framerate_t frame_rate_index;
     uint32_t num_packets, packet_size;
-    char *dma_device_file;
     int depth;
     dc1394framerates_t framerate_list;
     dc1394color_coding_t color_id;
@@ -3993,15 +3992,6 @@
     internal_status = camwire_bus_get_userdata(c_handle);
     ERROR_IF_NULL(internal_status);
 
-    if (cfg->dma_device_name[0] == '\0')
-    {
-	dma_device_file = 0;
-    }
-    else
-    {
-	dma_device_file = cfg->dma_device_name;
-    }
-    
     /* If dc1394_capture_stop() is called without a preceding
        successful call to dc1394_capture_setup(), libdc1394 used to get
        into a tangled state.  That is why we keep track with the
--- camwire2-1.9.0-Source/examples/measureconf_1394.c.orig	2011-02-17 03:31:13.000000000 +0100
+++ camwire2-1.9.0-Source/examples/measureconf_1394.c	2011-08-06 18:20:13.159740017 +0200
@@ -621,7 +621,6 @@
 static int setup_trigger(const Camwire_handle c_handle, const int serial_dev)
 {
     int result, ext_trig;
-    double trig_time;
     void *buffer;
 
     single_shot_capable = 1;  /* True unless we find out otherwise.*/
@@ -633,7 +632,7 @@
     {
 	camwire_set_trigger_polarity(c_handle, 1);
 	camwire_set_run_stop(c_handle, 1);  /* Camera waits for a trigger.*/
-	trig_time = trigger(c_handle, serial_dev);
+	 trigger(c_handle, serial_dev);
 	complete_acquisition(c_handle);  /* Make sure it's arrived.*/
 	result = camwire_point_next_frame_poll(c_handle, &buffer, 0);
 	if (result != CAMWIRE_SUCCESS)
@@ -979,7 +978,7 @@
 		      const int serial_dev)
 {
     dc1394video_mode_t video_mode;
-    double framerate, shutter;
+    double framerate;
     double f_dummy;
     double T, trig_time, timestamp;
     double transmit_time_in_ts_calc;
@@ -1024,7 +1023,7 @@
 /* 	nanosleep(&nap, 0); */
 
 	/* Trigger: */
-	shutter = set_random_shutter(c_handle);
+	 set_random_shutter(c_handle);
 	if (ext_trig_OK)  trig_time = trigger(c_handle, serial_dev);
 	else              trig_time = soft_trigger(c_handle);
 /* 	fprintf(stderr, ".");  // Debugging. */
